src/components/MealsEntry/MealsEntry.spec.js

```diff
@@ -0,0 +1,21 @@
+import { render, screen } from '@testing-library/react'
+import MealEntry from './MealsEntry'
```

Comment:
Is there a 's' missing in the first 'MealEntry'?

Reply:
Not really. The component and the export function are named differently. I wanted to change that. Thanks for reminding me :-)